frame/support/test/tests/storage_layers.rs

```diff
@@ -0,0 +1,197 @@
+// This file is part of Substrate.
+
+// Copyright (C) 2020-2022 Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use frame_support::{
+	assert_noop, assert_ok,
+	dispatch::{DispatchError, DispatchResult},
+	storage::with_storage_layer,
+	StorageMap, StorageValue,
+};
+use sp_io::TestExternalities;
+
+pub trait Config: frame_support_test::Config {}
+
+frame_support::decl_module! {
```

Comment:
Is this a V1 module just to test that it also works with the only syntax?

Reply:
updated it